tools/llama-bench/llama-bench.cpp

```diff
@@ -323,7 +347,7 @@ static void print_usage(int /* argc */, char ** argv) {
     printf("\n");
     printf(
         "Multiple values can be given for each parameter by separating them with ',' or by specifying the parameter "
-        "multiple times.\n");
+        "multiple times. Ranges can be specified with 'start-end' or 'start-end+step'.\n");
```

Comment:
You should specify that the ranges are inclusive.

Reply:
I changed it to `first-last+step`. Is that clear enough?

Reply:
I guess that also works but I think it would make sense to also rename the variables in the code to match the help.